cmd/attest.go

```diff
@@ -156,6 +170,10 @@ func addKeyFlag(cmd *cobra.Command) {
 	cmd.PersistentFlags().StringVar(&key, "key", "AK", "indicates type of attestation key to use <gceAK|AK>")
 }
 
+func addTeeTechnology(cmd *cobra.Command) {
+	cmd.PersistentFlags().StringVar(&teeTechnology, "tee_technology", "", "indicates the type of TEE hardware <sev-snp>")
```

Comment:
make <sev-snp> text the same as the above error message about valid values.

Reply:
By default it is set to nil, if used then the possible value for now is `"sev-snp"` only.

Reply:
If user left the flag empty then it will say ,
`Error: flag needs an argument: --tee_technology`

Reply:
Should I change the description to `"indicates the type of TEE hardware, if set to sev-snp"`